pkg/relay/relay.go

```diff
@@ -0,0 +1,657 @@
+package relay
```

Comment:
Should not be needed.

Reply:
reference by the chain: sfu.go:session->publisher.go:relaypeer->relay.peer.